include/vcpkg/base/strings.h

```diff
@@ -3,7 +3,7 @@
 #include <vcpkg/base/lineinfo.h>
 #include <vcpkg/base/optional.h>
 #include <vcpkg/base/stringview.h>
-#include <vcpkg/base/to_string.h>
+#include <vcpkg/base/to-string.h>
```

Comment:
I understand we want consistency, but I liked the symmetry in the header name and the function it provides.

Reply:
I guess I think if a header is providing only one function that suggests that the header shouldn't exist :P